app/src/main/java/org/mozilla/focus/activity/MainActivity.java

```diff
@@ -801,7 +803,7 @@ public HomeFragment createHomeFragment() {
 
     public void sendBrowsingTelemetry() {
         final SafeIntent intent = new SafeIntent(getIntent());
-        if (intent.getBooleanExtra(EXTRA_TEXT_SELECTION, false)) {
+        if (intent.getBooleanExtra(AppLaunchMethod.EXTRA_TEXT_SELECTION, false)) {
```

Comment:
I think this is somewhat deprecated by the new sendLaunchTelemetry() @mxlius what do you think?